static_dns/dns.tf

```diff
@@ -0,0 +1,9 @@
+
+resource "cloudflare_record" "_checkdns" {
+  zone_id = data.cloudflare_zone.dns_zone.id
+  name    = "_checkdns"
+  value   = "127.0.0.1"
```

Comment:
It needs to have a value, as this is a dummy record `127.0.0.1` seems sensible?

Reply:
I guess that works fine, we can also use `1.1.1.1` if you like that more or `10.0.0.0`